massa-cipher/src/decrypt.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+use aes_gcm_siv::aead::{Aead, NewAead};
+use aes_gcm_siv::{Aes256GcmSiv, Key, Nonce};
+use massa_hash::Hash;
+
+use crate::error::CipherError;
+
+pub fn decrypt(password: &str, data: &[u8]) -> Result<Vec<u8>, CipherError> {
+    let cipher = Aes256GcmSiv::new(Key::from_slice(
+        Hash::compute_from(password.as_bytes()).to_bytes(),
+    ));
+    let mut iter = data.split(|c| c == &b':');
```

Comment:
no need for a separator, nonce is always 12 bytes (also there should be a NONCE_SIZE=12 constant)

Reply:
Done 👍